requirements/tests.rqr

```diff
@@ -1,3 +1,4 @@
 bert_score
 transformers
 sentence_transformers
+ibm-cos-sdk
```

Comment:
we test ibm-cos here? how?

Reply:
We test the API using mocks. See test_loaders.py.